float64[] joint_commands